createdisk.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
+set -exuo pipefail
+
```

Comment:
iirc this was causing some unexpected failures in the past, I fixed some of them, but I forgot which state we are in. Could you test this, or should we wait for CI?

Reply:
You caught me, I was planning to do CI Driven Development :)